### PR TITLE
D3ASIM-2964: Added check for price on the external strategies, bid/of…

### PR DESCRIPTION
**Pull request by spyrostz**
…fer price should always be positive.

### CODE REVIEW

File: tests/contracts/token_fixtures.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def ganachecli_command():
     if "GANACHE_BINARY" in os.environ:
         return os.environ["GANACHE_BINARY"], 5
     else:
-        return 'ganache-cli', 2
+        return 'ganache-cli', 8
```
Comment (faizan2590) [inline tests/contracts/token_fixtures.py:30]: I assume thats a mistake, you are not testing this in ganache :-)

**Review by faizan2590 — APPROVED** [review]
LGTM

**Review by hannesdiedrich — APPROVED** [review]
LGTM

File: tests/contracts/token_fixtures.py (hunk 1 above)
Comment (spyrostz) [reply tests/contracts/token_fixtures.py:30]: You are right, this should be reverted :)

**Pull request merged**